src/plugins/lua/antivirus.lua

```diff
@@ -23,46 +23,6 @@ local redis_params
 
 local N = "antivirus"
 
-if confighelp then
```

Comment:
Is there something that I can do right now, or can you doe it? If you look at my changes, they are really very little. In fact I kicked the trim() function and changed some lines concerning the 310 parsing. This is much cleaner than before. And hopefully the parsing is fixed now and forever ;-)